entity-framework/core/modeling/relationships/navigations.md

```diff
@@ -17,7 +17,7 @@ EF Core relationships are defined by [foreign keys](xref:core/modeling/relations
 
 ## Reference navigations
 
-Navigations come in two forms--reference and collection. Reference navigations are simple object references to another entity. They represent the "one" side(s) of [one-to-many](xref:core/modeling/relationships/one-to-many) and [one-to-one](xref:core/modeling/relationships/one-to-one) relationships. For example:
+Navigations come in two forms — reference and collection. Reference navigations are simple object references to another entity. They represent the "one" side(s) of [one-to-many](xref:core/modeling/relationships/one-to-many) and [one-to-one](xref:core/modeling/relationships/one-to-one) relationships. For example:
```

Comment:
If we do this, we should probably use the HTML entity instead (assuming it's supported)
```suggestion
Navigations come in two forms&mdash;reference and collection. Reference navigations are simple object references to another entity. They represent the "one" side(s) of [one-to-many](xref:core/modeling/relationships/one-to-many) and [one-to-one](xref:core/modeling/relationships/one-to-one) relationships. For example:
```

Reply:
Yuck :( As this is markdown shouldn't we be able to just user a plain regular dash and have it converted (or whatever is needed) when that gets rendered as HTML?